src/main/kotlin/com/example/demo/controller/TodoController.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.demo.controller
+
+import com.example.demo.controller.dto.TodoInsertRequest
+import com.example.demo.model.TodoModel
+import com.example.demo.service.TodoService
+import org.springframework.web.bind.annotation.*
+
+//@Controller
+@RestController
+@RequestMapping("todo")
+class TodoController(
+    private var todoService: TodoService
+    ) {
+
+    @GetMapping("/list")
+    fun index(): List<TodoModel>  {
+        println("www")
+//        val modelList = sampleService.find()
+        return todoService.find()
+    }
+
+    @PostMapping("/insert")
+    fun insertData(@RequestBody body:TodoInsertRequest): Boolean {
```

Comment:
APIのレスポンスとしてBoolean単体を返すことは殆どないです
```suggestion
    fun insertData(@RequestBody body:TodoInsertRequest) {
```